setup.cfg

```diff
@@ -5,7 +5,7 @@ exclude=docs
 [flake8]
 line_length=88
 exclude=docs,tests
-ignore=E203, E501, W503, D
+ignore=E203, E501, E800, W503, D
```

Comment:
`E800` is most of the point to flake8-eradicate. It looks like we have a mix of truly commented out code and some things like test suite code with comments showing values which happen to be valid Python. Think we should just defer eradicate for now?

Reply:
removed flake8-eradicate